docs/data/sql_functions.yml

```diff
@@ -377,6 +377,12 @@ string:
   - sql: SUBSTR(string, integer1[, integer2])
     table: STRING.substr(INTEGER1[, INTEGER2])
     description: Returns a substring of string starting from position integer1 with length integer2 (to the end by default).
+  - sql: JSON_QUOTE(string)
```

Comment:
One thing I forgot to mention: also need to update chinese doc.
It's ok to put there english version of description however it's better to keep them synced

Reply:
Created JIRA https://issues.apache.org/jira/browse/FLINK-35800 (and added a comment to original JIRA https://issues.apache.org/jira/browse/FLINK-34111)

Reply:
Thanks for creation of jira
I would still vote for having filled chinese doc with english description which then could be translated within the jira issue you've created